django/test/runner.py

```diff
@@ -581,6 +581,22 @@ def add_arguments(cls, parser):
             ),
         )
 
+    def log(self, msg, level=None):
+        """Log the given message at the given logging level."""
+
+        if level is None:
+            level = logging.INFO
+
+        if self.verbosity == 0:
+            return
```

Comment:
You can simplify the rest of the method by doing:

```python
if self.verbosity == 1 and level is not None and level < logging.INFO:
    return
print(msg)
```

(This is what I meant when I said the `logging_level` variable could be eliminated.)

Reply:
We can club the `self.verbosity == 0` with this as well. Something like this
```
def log(self, msg, level=None):
    """Log the given message at the given logging level."""
    if self.verbosity == 0 or (
        self.verbosity == 1 and level is not None and level < logging.INFO
    ):
        return
    print(msg)
```

Reply:
Yeah, that works, too.